src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -58,17 +58,17 @@ public PersonCard(Person person, int displayedIndex) {
                 .sorted(Comparator.comparing(skill -> skill.skillName))
                 .forEach(skill -> skills.getChildren().add(new Label(skill.skillName)));
 
-        String jobIdentifier = person.getMatch();
-        if (jobIdentifier != null) {
+        String jobIdentifier = person.getMatch().orElse("Unemployed");
```

Comment:
Seem like a redundant step, could just check isEmpty I think.

Reply:
getMatch() returns an Optional<String>, I have to get the value out via the `orElse` since Optional.empty().get() returns an exception. The alternative is having a separate check with `person.isMatchPresent` then safely calling the get() method.

Reply:
Yes, we should do a separate check. 

I think this line of code could be considered confusing too, since technically, "Unemployed" is not a jobId.

Reply:
Resolved.

Reply:
By fixing the issue below, this is resolved.